src/firefox/index.coffee

```diff
@@ -56,7 +56,7 @@ pageMod.PageMod {
 # codesy home page
 pageMod.PageMod {
   include : [
-    /.*localhost.*/
+    /.*127.0.0.1:8443.*/
```

Comment:
This file and the firefox package.json are meant to stay production ready.

You can change the dev_domain variable in the gulp file to achieve this.